### PR TITLE
Add `timeFilterByColumn` macro

### PR DESCRIPTION
**Pull request by simPod**
Followup from https://github.com/Vertamedia/clickhouse-grafana/pull/68

### CODE REVIEW

File: dist/sql_query.ts [hunk 1]
```diff
@@ -102,16 +90,69 @@ export default class SqlQuery {
...
+        let to = SqlQuery.convertTimestamp(SqlQuery.round(range.to, round || 0));
+
+        // Extend date range to be sure that first and last points
+        // data is not affected by round
+        if (round > 0) {
+            to += (round * 2) - 1;
+            from -= (round * 2) - 1
+        }
+
+        console.log(query
```
[the remark this answers is not in the document]
Comment (simPod) [reply dist/sql_query.ts]: Forgot to run `build`. Will be gone, thx

**Comment by simPod** [comment]
I have covered additions to sql_query. Is that k?

File: specs/sql_query_specs.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { describe, expect, it } from './lib/common';
+import SqlQuery, { TimeRange } from '../src/sql_query';
+import moment from "moment";
+import { RawTimeRangeStub } from './lib/raw_time_range_stub';
+
+describe("Query SELECT with $timeFilterByColumn and range with from and to:", () => {
+  const query = "SELECT * FROM table WHERE $timeFilterByColumn(column_name)";
+  const range: TimeRange = {
+    from: moment('2018-12-24 01:02:03'),
```
Comment (hagen1778) [inline specs/sql_query_specs.ts]: `moment` value will vary depending on timezone, so those tests will fail if run in different tz. I suppose, it should be converted to particular tz or value in `expect` should be generated from `moment`

**Comment by hagen1778** [comment]
The error could be caught easily if there would be Travis running karma tests, but, unfortunately, I can't enable it for this repo. Sorry about that

**Comment by simPod** [comment]
@hagen1778 BTW why it cannot be enabled? I can try to setup it in later PR.

**Comment by hagen1778** [comment]
Lack of permissions to setup integration with Travis.

**Comment by simPod** [comment]
Isn't there anyone with petmissions? ;) Would be a great addition I suppose.

BTW added UTC tz to moments

**Comment by hagen1778** [comment]
Thank you for contribution!

**Pull request merged**